KappaAnalysis/interface/KappaEventProviderBase.h

```diff
@@ -36,7 +36,8 @@ class KappaEventProviderBase: public EventProviderBase<TTypes> {
 		m_fi.SpeedupTree(128*1024*1024); // in units of bytes
 
 		// auto-delete objects when moving to a new object. Not default root behaviour
-		m_fi.eventdata.SetAutoDelete(true);
```

Comment:
From [this thread](https://root.cern.ch/root/roottalk/roottalk01/4023.html) in the RootTalk I understand, that this call is not need any more since long time. So it might be good to remove it. Could the fact, that you spot this now, be related to ROOT 6?